src/dbt_score/scoring.py

```diff
@@ -19,6 +20,10 @@ class Score:
     value: float
     badge: str
 
+    def __post_init__(self) -> None:
+        """Auto-round score down to 1 decimal place."""
+        self.value = math.floor(self.value * 10) / 10
```

Comment:
Instead of overriding the provided score upon instantiation, I would suggest creating a separate `@property`, i.e. a derived value. This would allow:
- Keeping the original value
- Keeping things correct when mutated (e.g. `score = Score(9.99, 'foo'); score.value = 5`)

WDYT?

Reply:
Yep, that's definitely also something that can work.

Would you then suggest we only use the property (e.g. `rounded_value`) in the plain (human readable) output?

Reply:
Yes, and maybe keep the non-rounded in the JSON output? Although we could also use the rounded one in JSON, for consistency purposes 🤔

Reply:
Let's keep the non-rounded in the machine-readable outputs 👍

Reply:
I've updated the PR to reflect this.